builtin/game/misc.lua

```diff
@@ -234,8 +234,14 @@ else
 	end
 
 	function core.colorize(color, message)
-		return core.get_color_escape_sequence(color) .. message .. core.get_color_escape_sequence("#ffffff")
+		local output = ""
+		for line in message:gmatch("([^\n]+)") do
```

Comment:
The brackets aren't needed. `gmatch` produces the whole match if no captures are provided anyway.